web/angular-wallet/src/app/setup/account/create-passive/create-passive.component.html

```diff
@@ -1,17 +1,9 @@
 <div class="address-container">
-    <mat-form-field>
-        <input
-          matInput
-          prefix="BURST-"
-          burstInputValidator
-          [patterns]="burstAddressPattern"
-          mask="____-____-____-_____"
-          [dropSpecialCharacters]="false"
-          [showMaskTyped]="true"
-          [(ngModel)]="address"
-          name="address"
-          placeholder="{{ 'address' | i18n }}"
-        >
-        <mat-hint>Type your new burst address</mat-hint>
-    </mat-form-field> <i class="fa fa-lg fa-fw fa-arrow-right" (click)="submit(this.address)"></i>
-</div>
+  <mat-form-field appearance="outline">
```

Comment:
Cool. I had this on my radar already. @mahenrique94 and me saw this during our hackathon